### ISSUE TITLE
inference bug?

### ISSUE THREAD

**Issue by muye5** [hankcs/LDA4j#1]
phi never be used in inference?

**Comment by hankcs** [comment]
Yes, as I mentioned in the [comment](https://github.com/hankcs/LDA4j/blob/master/src/test/java/com/hankcs/lda/TestCorpus.java), it is not so much a bug as a TODO feature. This method is under construction, but I did not have a good understanding of that paper. Anyway, I'll leave this issue opened until anyone(include myself) complete it.

**Comment by laoyang945** [comment]
Hi guys, we should use phi[k][doc[n]] to replace (nw[doc[n]][k] + beta) / (nwsum[k] + V \* beta) in the inference part.
